src/app/modules/main/chat_section/module.nim

```diff
@@ -620,6 +621,8 @@ method onContactUnblocked*(self: Module, publicKey: string) =
   self.view.chatsModel().blockUnblockItemOrSubItemById(publicKey, blocked=false)
 
 method onContactDetailsUpdated*(self: Module, publicKey: string) =
+  if(self.controller.isCommunity()):
```

Comment:
@endulab there is not point to do any updates on any other but the Chat section when contact get updated since contact participate only in 1:1 chats which are available in Chat section only.